MusicApp.Identity.Web/Controllers/IdentityController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using MusicApp.Identity.Application.Services.Interfaces;
+using MusicApp.Identity.Application.DTOs;
+
+namespace MusicApp.Identity.Web.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class IdentityController : ControllerBase
+{
+    private readonly IIdentityService _identityService;
+    private readonly string _secretKey;
+
+    public IdentityController(IIdentityService identityService, IConfiguration configuration)
+    {
+        _identityService = identityService;
+        _secretKey = configuration.GetSection("JWT:Key").Value;
```

Comment:
not controller action

Reply:
I dont understand(

Reply:
it is not responsibility of controller to get secretKey